WordPress/src/main/res/values/styles.xml

```diff
@@ -102,6 +102,7 @@
         <item name="android:statusBarColor">@android:color/transparent</item>
         <item name="android:windowIsFloating">false</item>
         <item name="android:fitsSystemWindows">true</item>
+        <item name="android:windowSoftInputMode">adjustResize</item>
```

Comment:
This was added because the keyboard was covering the `bottomsheet` this allows the sheet to readjust when the keyboard is opened.

Reply:
Looks good, we just need to re-test the sign-up bottom sheet and especially the Reader-filter bottom sheet.

Reply:
Thanks for pointing that out. I will put a card on the project board to track this. I appreciate your keen eye 👍